### PR TITLE
RDS instance generation of ec2

### PR DESCRIPTION
**Pull request by fatima99s**

### CODE REVIEW

**Review by mehakseedat63 — APPROVED** [review]

**Review by mehakseedat63 — CHANGES_REQUESTED** [review]
Fix linting issues

File: plugins/aws/rds/rdsInstanceGeneration.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                    helpers.addResult(results, 0, 'RDS instance is using current generation of EC2: ' + dbInstanceClass, region, Rds.DBInstanceArn);
+                }
+            });
+
+            rcb();    
+        },function() {
+            callback(null, results, source);
+        });
+    }
+}
```
Comment (fatima99s) [inline plugins/aws/rds/rdsInstanceGeneration.js:62]: };

Comment (mehakseedat63) [inline plugins/aws/rds/rdsInstanceGeneration.js:62]:
```suggestion
};
```

Comment (fatima99s) [inline plugins/aws/rds/rdsInstanceGeneration.js]: };

**Review by mehakseedat63 — APPROVED** [review]

**Review by mehakseedat63 — APPROVED** [review]

**Pull request merged**